group_vars/customer/campus/main.yml

```diff
@@ -1,6 +1,6 @@
 # Variables specific to the campus customer
-project_display_name: "FUN Campus ({{ env_type }})"
-project_description: "SPOC for universities ({{ env_type }})"
+project_display_name: "FUN Campus"
+project_description: "SPOC for universities"
```

Comment:
IMO there is a confusion between the OpenShift and the application layer. In an OpenShift context I want to get the `env_type`. I think, this should be fixed in the application layer (set the `SITE_NAME` to `{{ customer }}` for example). Do you read me?